doc/web-socket.rst

```diff
@@ -427,9 +427,25 @@ Broker encodes Zeek events as nested vectors using the following structure:
 ``args``
   Contains the arguments for the event in the form of another ``vector``.
 
+``metadata``
+  Contains a ``vector`` of key-value pairs (represented as further ``vectors``
+  of size 2) for which the first element is a ``count`` for identification
+  purposes and the second element any supported Broker data type. This vector
+  can be used to attach arbitrary metadata to events.
+
+  Starting with version 6.0, all events published by Zeek will contain
+  ``NetworkTimestamp`` identified by a count of ``1`` as metadata. The
+  data type for the value is a ``timestamp``.
```

Comment:
It's a big ambiguous whether version 6 refers to Broker or Zeek. Maybe like this?

```suggestion
  Zeek version 6.0 and up always includes the network time of an event as metadata.
  The key for a network timestamp is ``1`` and the data type for the value is a ``timestamp``.
```